Core/src/org/sleuthkit/autopsy/centralrepository/datamodel/CorrelationAttributeUtil.java

```diff
@@ -158,6 +192,51 @@ private static String getEmailAddressAttrDisplayName() {
         return correlationAttrs;
     }
 
+    /**
+     * Makes a correlation attribute instance from a phone number attribute of an
+     * artifact.
+     *
+     * @param artifact An artifact with a phone number attribute.
+     *
+     * @return The correlation instance artifact or null, if the phone number is
+     *         not a valid correlation attribute.
+     *
+     * @throws TskCoreException     If there is an error querying the case
+     *                              database.
+     * @throws CentralRepoException If there is an error querying the central
+     *                              repository.
+     */
+    private static void makeCorrAttrFromArtifactPhoneAttr(List<CorrelationAttributeInstance> corrAttrInstances, BlackboardArtifact artifact) throws TskCoreException, CentralRepoException {
```

Comment:
Just FYI, this is a revert of previous changes.

Reply:
I know this is simply reverting to what 'was'...

But we could have email address attributes in the CONTACT_ARTIFACT, which should produce an CR attribute.  Perhaps we should make a story to come back and revisit this later.

Reply:
I'd like us to make the fix now.

Reply:
Yes, it can have email, email_to, email_cc, email_bcc, phone, phone_to, phone_from, phone_home, phone_office, phone_mobile. I looked at the code and the same logic applies to some other artifacts. I think creating a story would be a good idea.

Reply:
@dannysmyda, please create the story, assign it to yourself, etc. Thanks!

Reply:
I'll just do it here then, for the phone attr method.